### ISSUE TITLE
Support for "sufix" buttons

### ISSUE THREAD

**Issue by StokeMasterJack** [ellipticaljs/paper-autocomplete#19]
paper-input allows you to attach a button to the input:

```
<paper-input label="total">
  <paper-icon-button suffix icon="search"></paper-icon-button>
</paper-input>
```

This would be a nice feature for paper-autocomplete.

**Comment by tachyon1337** [comment]
I agree...I will look to add it with the next minor version.
Thanx!

**Comment by ainvaltin** [comment]
The issue is closed but I just tested with ver 3.0 and it doesn't work (suffiks / prefix button is not visible inside paper-autocomplete). So is it a regression or...?

**Comment by jhuesos** [comment]
I am not sure... I need to test. Please reopen it and provide information
about how to reproduce it.

**Comment by ainvaltin** [comment]
About reproducing it - in the (Polymer 2 component's) html I have

    <paper-input>
        <paper-icon-button slot="prefix" icon="search" on-click="boom"></paper-icon-button>
    </paper-input>

    <paper-autocomplete>
        <paper-icon-button slot="prefix" icon="search" on-click="boom"></paper-icon-button>
    </paper-autocomplete>

the `paper-input` has a button in the left end while `paper-autocomplete` does not.

I don't have right to reopen the issue (I'm not the original reporter), please do it yourself. 
BTW I noticed that @tachyon1337 has closed the issue (almost) on the same time he commented about fixing it in the next minor version - it seems to me that he accidentally closed it when posting his comment and thus forgot about it?

**Comment by jhuesos** [comment]
I just reopened for you. I will take a look.

**Comment by jhuesos** [comment]
I have a working version of the fix. I will try to release it tomorrow

![2017-08-21_17-57-57](https://user-images.githubusercontent.com/339609/29527839-542fba6c-869a-11e7-9022-d86a08685443.png)

**Comment by jhuesos** [comment]
Fixed in v3.0.2
